tools/plugins/initial-config/src/main/kotlin/net/corda/cli/plugin/initialconfig/CryptoConfigSubcommand.kt

```diff
@@ -107,29 +97,37 @@ class CryptoConfigSubcommand : Runnable {
     var type: SecretsServiceType = SecretsServiceType.CORDA
 
     override fun run() {
-        val wrappingKeys = (1..numberOfUnmanagedWrappingKeys).toList().map { index ->
-            val (wrappingPassphraseSecret, wrappingSaltSecret) = createWrappingPassphraseAndSaltSecrets(index)
-            KeyDerivationParameters(
-                wrappingPassphraseSecret.root(),
-                wrappingSaltSecret.root()
-            )
+        if (type == SecretsServiceType.CORDA) {
+            checkParamPassed(passphrase)
+            { "'passphrase' must be set for CORDA type secrets." }
```

Comment:
nit (formatting):
```suggestion
            checkParamPassed(passphrase) { "'passphrase' must be set for CORDA type secrets." }
```

Same on lines below (103-104, 106-107)